sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/LogsQueryAsyncClient.java

```diff
@@ -178,7 +213,28 @@
      *
      * <p><strong>Execute a batch of logs queries</strong></p>
      *
-     * {@codesnippet com.azure.monitor.query.LogsQueryAsyncClient.queryBatch#LogsBatchQuery}
+     * <!-- src_embed com.azure.monitor.query.LogsQueryAsyncClient.queryBatch#LogsBatchQuery -->
```

Comment:
I should know this but ....

Did you have to manually add this to each method or did the javadoc build step did this automatically?

Reply:
The tags (`<!-- src_embed`) have to be updated manually (IntelliJ's find and replace helped) and then when you run mvn install, it should inject the code snippets.